common/app/common/configuration.scala

```diff
@@ -136,17 +144,7 @@ class GuardianConfiguration extends Logging {
       configuration.getIntegerProperty("tag_indexes.rebuild_rate_in_minutes").getOrElse(60)
   }
 
-  object environment {
-    private val installVars = new File("/etc/gu/install_vars") match {
-      case f if f.exists => IOUtils.toString(new FileInputStream(f))
-      case _ => ""
-    }
-
-    private val properties = Properties(installVars)
-
-    def apply(key: String, default: String) = properties.getOrElse(key, default).toLowerCase
-
-    val stage = apply("STAGE", "unknown")
+  object environment extends InstallVars.InstallationVars {
```

Comment:
Thanks for the feedback, John! One thing that's not quite clear to me is whether `stage` really is the only install variable that also belongs in the environment. If that's the case I'm happy to expose it as a val. Otherwise, I think it's a bit ad hoc